fuel-core-interfaces/src/p2p.rs

```diff
@@ -8,23 +8,54 @@ use crate::{
     model::ConsensusVote,
 };
 use async_trait::async_trait;
-use std::sync::Arc;
+use serde::{
```

Comment:
serde is an optional dependency, and usage needs to be under a `#cfg(feature = "serde")` attribute.